src/stmem.cc

```diff
@@ -61,7 +61,7 @@ mem_hdr::freeContent()
 {
     nodes.destroy();
     inmem_hi = 0;
-    debugs(19, 9, HERE << this << " hi: " << inmem_hi);
+    debugs(19, 9, this << " hi: " << inmem_hi);
```

Comment:
This method is not a constructor or destructor. It is used by the destructor. This code definitely needs some work outside this PR scope.